<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>584</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Carlo Nicolini carlo.nicolini@iit.it - Carlo Fantoni carlo.fantoni@iit.it</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="StimulusWidget" name="stimWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1024</width>
          <height>1024</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelParameters">
          <property name="text">
           <string>Parameters</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Slant °</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxSlant">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <double>-90.000000000000000</double>
            </property>
            <property name="maximum">
             <double>90.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>5.000000000000000</double>
            </property>
            <property name="value">
             <double>45.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Tilt °</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxTilt">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <double>270.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>90.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Def</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxDef">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <double>-5.000000000000000</double>
            </property>
            <property name="maximum">
             <double>5.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.400000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>StimSize</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxStimSize">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>80.000000000000000</double>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>NPoints</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="spinBoxNPoints">
            <property name="minimum">
             <number>50</number>
            </property>
            <property name="maximum">
             <number>2500</number>
            </property>
            <property name="singleStep">
             <number>50</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Period</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxPeriod">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>20.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>10.000000000000000</double>
            </property>
            <property name="value">
             <double>80.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Amplitude</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxAmplitude">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>50.000000000000000</double>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>25.000000000000000</double>
            </property>
            <property name="value">
             <double>125.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Stimulus Onset</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QComboBox" name="comboBoxDirections">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Low to high</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>High to low</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Left to right</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Right to left</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Stimulus motion</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QComboBox" name="comboBoxStimMotion">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <item>
             <property name="text">
              <string>Sawtooth</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Triangle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sinusoidal</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="11" column="0" colspan="2">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>FocalDistance</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxFocalDistance">
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-800.000000000000000</double>
            </property>
            <property name="maximum">
             <double>-400.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>5.000000000000000</double>
            </property>
            <property name="value">
             <double>-418.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Backprojection</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <widget class="QCheckBox" name="checkBoxBackProjection">
            <property name="text">
             <string>Active</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="14" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Def correction</string>
            </property>
           </widget>
          </item>
          <item row="14" column="1">
           <widget class="QCheckBox" name="checkBoxConstantDef">
            <property name="text">
             <string>Constant def mode</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="15" column="1">
           <widget class="QCheckBox" name="checkBoxDrawGrid">
            <property name="text">
             <string>Draw grid overlay</string>
            </property>
           </widget>
          </item>
          <item row="16" column="1">
           <widget class="QCheckBox" name="checkBoxCorrectShear">
            <property name="text">
             <string>Correct shear</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="17" column="1">
           <widget class="QCheckBox" name="checkBoxTranslation">
            <property name="text">
             <string>Translation</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="18" column="0">
           <widget class="QCheckBox" name="checkBoxCircularMask">
            <property name="text">
             <string>Circular Mask</string>
            </property>
           </widget>
          </item>
          <item row="18" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxMaskRadius">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>10.000000000000000</double>
            </property>
            <property name="value">
             <double>50.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="19" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Rand.motion</string>
            </property>
           </widget>
          </item>
          <item row="19" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxRandomMotion"/>
          </item>
          <item row="20" column="0">
           <widget class="QToolButton" name="pushButtonSaveToDir">
            <property name="text">
             <string>Set frames dir ...</string>
            </property>
           </widget>
          </item>
          <item row="20" column="1">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Saving frames</string>
            </property>
           </widget>
          </item>
          <item row="21" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQui">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuQui"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StimulusWidget</class>
   <extends>QWidget</extends>
   <header>StimulusWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
